features/home_page/map.feature

```diff
@@ -63,4 +63,10 @@ Feature: Map of local charities
     Given I visit the home page
     And I click "Close"
     Then I should see "Details updated by the organisation within the last 12 months"
-    Then I should see "Details NOT updated by the organisation within the last 12 months"
\ No newline at end of file
+    Then I should see "Details NOT updated by the organisation within the last 12 months"
+
+  Scenario: Do not showing meaning of large map icons
```

Comment:
I would change this scenario name to: "Do not show meaning of large map icons on volunteer ops page"